src/ProductConstructionService/ProductConstructionService.Api/Queue/PcsJobsProcessorStatus.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ProductConstructionService.Api.Queue;
+
+public enum PcsJobsProcessorState
```

Comment:
I'd get more generic and go `ServiceState`.

Then the statuses would be `Active`, `Stopping` and `Stopped`.
We could then document what all these mean here in the enum class

Reply:
I think `ServiceState` is a bit confusing, as it's only describing what the JobProcessor is doing. The other part of the service, the api, is always listening and working